nixos/modules/services/hardware/sane.nix

```diff
@@ -4,7 +4,11 @@ with lib;
 
 let
 
-  pkg = pkgs.sane-backends;
+  pkg = pkgs.sane-backends.override
+    (lib.optionalAttrs config.hardware.sane.drivers.scanSnap.enable {
+      scanSnapDriversUnfree = true;
```

Comment:
```suggestion
      scanSnapDriversUnfree = config.hardware.sane.drivers.scanSnap.enable;
```

Reply:
Hrm I can just leave out the optionalAttrs now that I think about it. Good catch.

Reply:
Oh, I missed those. I guess you could also leave it as is.

Not sure if we can have a tiny performance gain by only overriding if unfree, but it's the module system, so I'd say nicer code is more important.